src/kohlrahbi/__init__.py

```diff
@@ -102,8 +103,8 @@ def load_all_known_pruefis_from_file(
         click.secho(f"There is no 'content' section in the toml file: {path_to_all_known_pruefis}", fg="red")
         raise click.Abort()
 
-    pruefis: list[str] = content_section.get("pruefidentifikatoren")
-    return pruefis
+    pruefi_to_file_mapping: dict[str, str | None] = content_section.get("pruefidentifikatoren")
+    return pruefi_to_file_mapping
```

Comment:
wenn wir die section in der toml Datei gleich `pruefidentifikatoren` nennen können wir uns die einen der zwei get befehle in Zeile 97 und 106 sparen oder?